packages/debugger/lib/data/sagas/index.js

```diff
@@ -1007,14 +1043,13 @@ function literalAssignments(
   }
 
   let assignment = makeAssignment(
-    inModifier
-      ? {
-          compilationId,
-          astRef: node.id,
-          stackframe: currentDepth,
-          modifierDepth
-        }
-      : { compilationId, astRef: node.id, stackframe: currentDepth },
+    {
+      compilationId,
+      internalFor,
```

Comment:
This pattern seems like it's getting out of hand. Is there a way we could make this kind of referencing more cohesive? Can we define some type that houses all relevant fields for such a lookup?

Reply:
Handled now by `sourceId`!